src/components/Provider/Provider.tsx

```diff
@@ -25,12 +41,18 @@ class Provider extends React.Component<any, any> {
         }),
       }),
     ),
-    siteVariables: PropTypes.object,
-    componentVariables: PropTypes.object,
-    staticStyles: PropTypes.arrayOf(
-      PropTypes.oneOfType([PropTypes.string, PropTypes.object, PropTypes.func]),
-    ),
-    rtl: PropTypes.bool,
+    theme: PropTypes.shape({
```

Comment:
Is this required?  It looks to be based on the interface.

Reply:
Currently, docs are still generated from propTypes.  That needs to be moved to being generated from typings.  Then we can consider removing propTypes.

The other consideration is that most consumers are not running typescript.  So, I think we may end up keeping both interfaces and propTypes around.  Also, some runtime checks are really handy since we can do things with values (suggest icon names, etc).

Reply:
Sorry, I meant to say "should the `theme` propType be `.isRequired`?". I wasn't talking about the propTypes themselves.

Reply:
Got it, thanks.  We have a scheduled task to make a default theme.  For now, it is required.